.github/workflows/ios-ci.yml

```
@@ -0,0 +1,218 @@
+          echo " • Workspace:  ${XCWORK:-<none>}"
+
+      - name: Patch Podfile project path if needed
+        shell: bash
+        run: |
+          set -euo pipefail
+          PODFILE="${{ steps.ios_paths.outputs.PODFILE }}"
+          PODDIR="${{ steps.ios_paths.outputs.PODDIR }}"
+          XCPROJ="${{ steps.ios_paths.outputs.XCPROJ }}"
+
+          RELPROJ=$(python3 - <<'PY'
+import os
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 94: could not find expected ':'

(syntax-check)